### ISSUE TITLE
Wikiman errors on any command

### ISSUE THREAD

**Issue by skmd0** [filiparag/wikiman#17]
I am using ZSH shell on Arch Linux, Konsole terminal emulator. I have installed the package from the AUR. I have arch-wiki-docs installed also.

```sh
$ wikiman -S
/usr/bin/env: ‘dash’: No such file or directory
```

Available shells:

```sh
$ chsh --list-shells 
/bin/sh
/bin/bash
/usr/bin/git-shell
/bin/zsh
/usr/bin/zsh
```

**Comment by filiparag** [comment]
Thanks for reporting this error. I forgot to change shebang interpreter back to `/bin/sh`.
I will release updated version shortly, but until then download `dash` shell or use a temporary fix:
```bash
sudo sed -i '/^#!.*dash/d' /usr/bin/wikiman
```

**Comment by filiparag** [comment]
Issue is resolved in version [2.10](https://github.com/filiparag/wikiman/releases/tag/2.10).
